tests/Smoke/StdinTest.php

```diff
@@ -46,7 +46,7 @@ public function testFixingStdin()
         );
         static::assertSame($expectedError, $stdinResult->getError());
 
-        $path = str_replace('/', \DIRECTORY_SEPARATOR, basename(realpath($cwd)).'/'.$inputFile);
```

Comment:
isn't this change braking the usage of fixing the relatives and symlink/linked files?

maybe it's worth to add a test cases like those

Reply:
I'm not sure I understand what you mean. This is a test and the change adjust what we expect from stdout.